example_notebooks/7_ALS_toyblock_demo.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>This results in three flight lines. Since the scanner does not record any data between the second and the third leg (<code>active="false"</code>), there are only two flight strips.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>